packages/manager/modules/account-sidebar/src/user-infos/template.html

```diff
@@ -25,9 +25,15 @@
             ></span>
         </p>
         <p
+            data-ng-if="$ctrl.role !== 'provider'"
             class="manager-hub-user-infos_profile_link mb-1"
             data-ng-bind="$ctrl.getDisplayName()"
         ></p>
+        <p
+            data-ng-if="$ctrl.role === 'provider' && $ctrl.me.organisation"
```

Comment:
related to : https://github.com/ovh/manager/pull/6406/files?file-filters#r798573120